docs/source/user_guide/01_philosophy.rst

```diff
@@ -1,55 +1,7 @@
 
-The philosophy and the approach
-===============================
+The philosophy
```

Comment:
It's silly, but I'd prefer to leave it as "The philosophy and the approach" as it's actually an homage to a foundational work that we read in my class, and since that class inspired nbgrader, it's my own little way of recognizing that :-) This is the paper if you're curious:

http://web.stanford.edu/class/psych209a/ReadingsByDate/01_07/Marr82Philosophy.pdf